test/SpecHelper.hs

```diff
@@ -24,7 +24,8 @@ import Text.Heredoc
 
 import PostgREST.Config (AppConfig (..))
 import PostgREST.Types  (JSPathExp (..))
-import Protolude
+import Protolude        hiding (toS)
+import Protolude.Conv   (toS)
```

Comment:
So `Protolude.Conv.toS` is the legacy one right? I'll keep this in mind when working with the new Protolude.

If we later need to add utilitarian functions to workaround breaking changes, we can include them in https://github.com/PostgREST/postgrest/blob/master/src/PostgREST/Private/Common.hs.

Reply:
Right. It turns out that the legacy `toS` would throw exceptions when trying to decode broken UTF8 bytestrings. That risk is still there. Though unfortunately, the change also means that the non-legacy `toS` can't do the safe conversions between bytestrings or to bytestring. (See also https://github.com/protolude/protolude/issues/117.)